backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadService.java

```diff
@@ -49,6 +49,7 @@ public PatientBulkUploadResponse processPersonCSV(InputStream csvStream, UUID fa
     if (!errors.isEmpty()) {
       result.setStatus(UploadStatus.FAILURE);
       result.setErrors(errors.toArray(FeedbackMessage[]::new));
+      log.debug("CSV patient bulk upload rejected with the following errors: {}", errors);
```

Comment:
For my own understanding... you don't have to call `toString()` explicitly on `FeedbackMessage` to transform it into a string because of the `@ToString` annotation?? 🧐

Reply:
Not quite! But that's a reasonable guess 😄 Java automatically calls the `toString` function on any Object when trying to print it. I could've written `errors.toString()` on this line instead, but IntelliJ would've just put a little warning saying it's unnecessary code, since the Java compiler is smart enough to automatically call `toString` on print-like functions such as logging.

The `@ToString` Lombok annotation just makes the printed string readable/useful. [All Java Objects have a `ToString` method by default](https://docs.oracle.com/javase/7/docs/api/java/lang/Object.html#:~:text=toString()-,Returns%20a%20string%20representation%20of%20the%20object.,-void), but unless it's explicitly overridden somewhere Java will default to printing out the type of the object + the memory address it's stored in, which will give you something like `[FeedbackMessage@123459845]`. That is pretty useless, so I needed to override `ToString` to instead print out the information I cared about - in this case, the actual values of `scope`, `message`, and `indices`. I could've done that through a manual override, something like
```
@Override
public String toString() {
return
    "scope: " + this.scope + 
    "message: " + this.message + 
    "index: " + this.indices;
}
```
But on a hunch I thought Lombok might have a shortcut for this, since this is a really common use case and results in a lot of boilerplate. And they do! So adding the `@ToString` annotation transforms our printout from `[FeedbackMessage@123459845]` to `FeedbackMessage(scope=item, message=File is missing data in the race column., indices=[4]`.

Reply:
ah ha!!! this is a super helpful, detailed response to my question. I really appreciate it 😹 🤯

Reply:
Of course! Always happy to nerd out about Java 🤓